### PR TITLE
Added sidebar for Privacy page

### LINKED ISSUE #1312: Improvement in UI of Privacy page
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behavior:**

Currently `Privacy` page is not user friendly at all. We need to improve it by adding sidebar having links to different section.

**Expected behavior:**

We can expect something like Google `Privacy` page.

**Screenshots of the issue:**

**Susper**
screenshot_2018-12-19 susper - decentral search engine

**Google**
screenshot_2018-12-19 privacy policy privacy terms google

**Will work on the issue:**
Yes, I'm working on it

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
Fixes #1312 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:

- Added a sidebar for Privacy page 

- https://pr-1315-fossasia-susper.surge.sh

screenshot_2018-12-19 susper - decentral search engine 1

### CODE REVIEW

**Comment by shreyanshdwivedi** [comment]
@praveenojha33 @AakashMallik please review. Travis may fail as it is currently down in our repo.

**Comment by shreyanshdwivedi** [comment]
@simsausaurabh I've fixed the margins. However, the hovering color is because of some declarations in `base.css` which cannot be overwritten. 
Also, regarding implementation of this as a component, I don't think it's better idea because we only need it once. It's not reusable in any other part.
Also, we need similar thing for `Terms` page too, so it will be redundant if we opt for using two components having similar feature.
screenshot_2018-12-19 susper - decentral search engine

**Comment by simsausaurabh** [comment]
Please update the PR link.

**Comment by shreyanshdwivedi** [comment]
@simsausaurabh the surge link is updated. Please review :)

**Comment by shreyanshdwivedi** [comment]
@praveenojha33 @simsausaurabh I've made all the changes and also written test for increasing coverage. Please review :)

**Comment by shreyanshdwivedi** [comment]
@simsausaurabh @praveenojha33 I've made the required changes and made the `Privacy` and `Terms` page similar. Also, I've squashed the commits. Please review :)

**Comment by simsausaurabh** [comment]
It is still changing color on hovering. Did you update code?

**Comment by shreyanshdwivedi** [comment]
@simsausaurabh I checked it on firefox as well as Chrome. It is working fine for me on both
I think it is one of the glitches surge ave. It is not able to render css properly sometimes.
If you are checking the changes on your local, please let me know then. I am unable to reproduce it on local.
screenshot from 2018-12-26 20-03-04

**Comment by shreyanshdwivedi** [comment]
@praveenojha33 can you please check if the problem faced by @simsausaurabh is only due to surge or it is happening on local too. I checked and did not found any issue on my local.

File: src/app/privacy/privacy.component.css [hunk 1]
```diff
@@ -81,11 +116,25 @@ h2 {
...
+
+	.sidebar-list{
+		display: none;
+	}
+
+	.wrapper{
+		position: relative;
+	}
+
+  	.navbar-brand {
```
Comment (simsausaurabh) [inline src/app/privacy/privacy.component.css:131]: Always follow proper indentation throughout the codebase.

**Comment by shreyanshdwivedi** [comment]
@simsausaurabh @praveenojha33 please merge

**Comment by mariobehling** [comment]
Thanks. Please resolve conflicts.

**Comment by shreyanshdwivedi** [comment]
@mariobehling I've resolved the conflicts. Please review :)

**Comment by shreyanshdwivedi** [comment]
@praveenojha33 please review now.

**Pull request merged**